projects/dotcms-field-elements/src/components/dot-select/dot-select.tsx

```diff
@@ -0,0 +1,145 @@
+    @Prop() options: string;
+    @Prop() required: boolean;
+    @Prop() requiredmessage: string;
+    @Prop({ mutable: true }) value: string;
+
+    @Event() valueChange: EventEmitter;
+    @Event() statusChange: EventEmitter;
+
+    @State() _options: DotOption[];
+    @State() _valid = true;
+    _dotTouched = false;
+    _dotPristine = true;
+
+    componentWillLoad() {
+        this._options = getDotOptionsFromFieldValue(this.options);
+        this.emitInitialValue();
+        this.emitStatusChange();
+    }
+
+    hostData() {
+        return {
+            class: {
+                'dot-valid': this.isValid(),
+                'dot-invalid': !this.isValid(),
+                'dot-pristine': this._dotPristine,
+                'dot-dirty': !this._dotPristine,
+                'dot-touched': this._dotTouched,
+                'dot-untouched': !this._dotTouched
+            }
+        };
+    }
+
+    /**
+     * Reset properties of the field, clear value and emit events.
+     *
+     * @memberof DotSelectComponent
+     */
+    @Method()
+    reset(): void {
+        this._dotPristine = true;
+        this._dotTouched = false;
+        this._valid = true;
+        this.value = '';
+        this.emitInitialValue();
+        this.emitStatusChange();
+    }
+
+    render() {
+        return (
+            <Fragment>
+                <label htmlFor={this.name}>{this.label}</label>
+                <select
+                    class={this.getClassName()}
+                    id={this.name}
+                    disabled={this.shouldBeDisabled()}
+                    onChange={(event: Event) => this.setValue(event)}>
+
+                    {this._options.map((item: DotOption) => {
+                        return (
+                            <option
+                                selected={this.value === item.value ? true : null}
+                                value={item.value}
+                            >
+                                {item.label}
+                            </option>
+                        );
+                    })}
+
+                </select>
+                {this.hint ? <span class='dot-field__hint'>{this.hint}</span> : ''}
+                {!this.isValid() ? <span class='dot-field__error-message'>{this.requiredmessage}</span> : ''}
+            </Fragment>
+        );
+    }
+
+    private getClassName(): string {
+        return this.isValid() ? '' : 'dot-field__select--error';
+    }
+
+    private shouldBeDisabled(): boolean {
+        return this.disabled ? true : null;
+    }
+
+     // Todo: find how to set proper TYPE in TS
+    private setValue(event): void {
+        this._dotPristine = false;
+        this._dotTouched = true;
+        this.value = event.target.value;
+        this.emitValueChange();
+        this.emitStatusChange();
+    }
+
+    private emitInitialValue() {
+        if (!this.value) {
+            this.value = this._options[0].value;
+            this.emitValueChange();
```

Comment:
I thin this is the only one componente who emit the initial value, if we are doing this here we have to this in the others components

Reply:
no, this component is unique and when you reset it, it needs to behave differently from others, so you need to pick the first option of the dropdown and then emit it.

on the others `elements` the value is being set here https://github.com/dotCMS/core-web/blob/2d20298faddd03d29f2d1087cc8716c96c0369b3/projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx#L72